src/scripts/upgrader.zig

```diff
@@ -0,0 +1,141 @@
+/// Smoke test for upgrade procedure:
+/// - given two tigerbeetle binaries, old and new
```

Comment:
These would need to be two releases (ie, multiversion binaries) and not just the output from `zig build`, correct?

Reply:
Correct (though I am leaning that we should move multiversioning to `zig build` so that `zig build -Dmultiversion` produces one)! 

Clarified

Reply:
Re `zig build -Dmultiversion` - see also [this thread](https://tigerbeetle.slack.com/archives/C04375JUAMS/p1722902746155869) about packaging all this for Nix.